lib/src/kv.rs

```diff
@@ -20,6 +20,10 @@ pub enum KvAction {
     BeginTx,
     Commit { tx_id: u64 },
     Backup,
+    // Iterator operations
```

Comment:
Comments inside enums should be `///` docstrings -- please label each iter operation with an explanation of how to use it. It's not immediately obvious to me how it works.